include/materials/material_utility.h

```diff
@@ -0,0 +1,68 @@
+#ifndef MPM_MATERIAL_UTILITY_H_
+#define MPM_MATERIAL_UTILITY_H_
+
+#include <cmath>
+
+#include "Eigen/Dense"
+
+namespace mpm {
+namespace material_utility {
+
+//! Compute mean stress p
+//! \param[in] stress Stress in Voigt notation where positive is tension
+inline const double compute_mean_p(const Eigen::Matrix<double, 6, 1>& stress);
+
+//! Compute deviatoric stress
+//! \param[in] stress Stress in Voigt notation where positive is tension
```

Comment:
Please change this to positive is compression

Reply:
Wait, the input stress is always positive tension

Reply:
I'm really sorry, I got this mixed up. You are right, stresses we get are positive in tension. Let's be consistent and have mean stress be also positive in tension. Sorry about that!! I was incorrectly looking at the magnitude not a component.